media/js/src/levelsdashboard.tsx

```diff
@@ -32,7 +33,7 @@ export const LevelsDashboard: React.FC = () => {
         setapprenticeCount(apprenArr.length);
     }
 
-    const completedNovice = completionCount('Novice', questionsList);
+    const completedNovice = completionCount('Beginner', questionsList);
```

Comment:
Are we changing this back? I've changed this to Novice everywhere else. The localstorage setting for levels will be Novice now if you refresh the cache. We can change it back though but it has to be changed everywhere.